instrumentation/otel-webserver-module/docker/centos7/Dockerfile

```diff
@@ -0,0 +1,258 @@
+FROM centos:7
+
+ARG BUILD_NUMBER
+
+LABEL NAME=apm/build-image-webserver-agent-centos6-x64 VERSION=$BUILD_NUMBER
+
+ENV GOSU_ARCH amd64
+ENV JDK_ARCH x64
+
+# create default non-root user
+RUN groupadd -r swuser && useradd -u 1000 -g swuser -m -s /sbin/nologin -c "default non-root user" swuser
+
+RUN chmod -R 777 /opt
+
+# RUN curl https://www.getpagespeed.com/files/centos6-eol.repo --output /etc/yum.repos.d/CentOS-Base.repo
+
+# install required packages
+RUN yum update -y \
+    && yum install -y \
+    yum install wget -y \
+    yum install -y zlib-devel \
+    yum install -y openssl-devel \
+    yum install java-1.8.0-openjdk -y \
+    yum install pcre-devel -y \
+    yum install xz -y \
+    yum install httpd -y \
+    && yum clean all
+
+# install lcov package
+RUN wget https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
+	&& yum install ./epel-release-latest-*.noarch.rpm -y \
+	&& yum install lcov -y \
+	&& yum clean all
+
+# install devtoolset toolchain
+# RUN curl https://www.getpagespeed.com/files/centos6-eol.repo --output /etc/yum.repos.d/CentOS-Base.repo \
+#  && curl https://www.getpagespeed.com/files/centos6-epel-eol.repo --output /etc/yum.repos.d/epel.repo \
+  RUN yum -y install centos-release-scl \
+#  && curl https://www.getpagespeed.com/files/centos6-scl-eol.repo --output /etc/yum.repos.d/CentOS-SCLo-scl.repo \
+#  && curl https://www.getpagespeed.com/files/centos6-scl-rh-eol.repo --output /etc/yum.repos.d/CentOS-SCLo-scl-rh.repo \
+  && yum install yum-utils -y \
+  && yum-config-manager --enable rhel-server-rhscl-7-rpms -y \
+  && yum install devtoolset-7 -y \
```

Comment:
I think we should use the default compiler for better compatibility.We can only use grpc v1.33.2 and abseil 20200225.3 with gcc 4.8.

Reply:
Thanks @owent for this. This is very initial change and exploration is done to run webserver module on centos7.  Though this is not affecting, I feel this can be submitted for now.
The other two files are related to a bug and needs to closed.